src/Import/include/FMI3/fmi3_import_convenience.h

```diff
@@ -109,7 +111,21 @@ typedef struct {
 } fmi3_logger_context_t;
 
 /**
-    \brief Collect model information by counting the number of variables with specific properties and fillinf in fmi3_import_model_counts_t struct.
+ * \brief Create empty fmi3_import_model_counts_t struct.
+ * @returns  fmi3_import_model_counts_t struct with all fields set to 0.
+ *           Free via fmi3_import_free_model_counts(...)
+ */
+FMILIB_EXPORT
+fmi3_import_model_counts_t* fmi3_import_init_model_counts();
```

Comment:
Convenience functions. structs fields need to be initialized, they are not zero by default.

Reply:
Just set them to 0 in `fmi3_import_collect_model_counts`? No need for separate API to do that